### ISSUE TITLE
Automated coding style reformat?

### ISSUE THREAD

**Issue by marcan** [osxmidi/LinVst#146]
I've noticed that the LinVst code has quite random indentation, which makes it a bit hard to follow the code. Just wondering if you've considered running the LinVst code through an automated code formatting tool, like clang-format? I think that would make it easier for other people who might want to contribute to understand it.

If it's something you would be interested in, I can give it a shot and send a PR with the reformatting.

**Comment by osxmidi** [comment]
Sure, if you want to do it.

**Comment by keybreak** [comment]
@marcan can you please still do it? :upside_down_face:
